UPDATING.md

```diff
@@ -1090,12 +1101,12 @@ Operators involved:
   * GCP Function Operators
     * GcfFunctionDeployOperator
   * GCP Cloud SQL Operators
-    * CloudSQLCreateInstanceOperator
-    * CloudSQLInstancePatchOperator
-    * CloudSQLDeleteInstanceOperator
-    * CloudSQLCreateInstanceDatabaseOperator
-    * CloudSQLPatchInstanceDatabaseOperator
-    * CloudSQLDeleteInstanceDatabaseOperator
+    * CloudSqlInstanceCreateOperator
+    * CloudSqlInstancePatchOperator
```

Comment:
The Updating.md for the previous version should contain the original name that existed in that version.